src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/UnusedImportsCheck.java

```diff
@@ -109,6 +105,54 @@
  * <pre>
  * &lt;module name="UnusedImports"/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * import demo.Demo;  // violation, Unused import - demo.Demo
+ *
+ * import java.lang.String; // violation, Unused import - java.lang.String
+ *
+ * import java.util.Stack;  // OK
+ * import java.util.Map;   // violation, Unused import - java.util.Map
+ *
+ * import java.util.List; // OK
+ *
+ * &#47;&#42;&#42;
+ *  &#42; @link List
+ * &#42;&#47;
+ * class MyClass{
+ *  public static void main(String args[]){
+ *   Stack stack = new Stack();
```

Comment:
Remove main method and keep only stack as field.

Reply:
done